docs/sms-send-how-to.md

```diff
@@ -42,6 +44,10 @@ grpcurl \
   messaging.MessageCore.SendTextMessage
 ```
 
+### Send binary messages
```

Comment:
Since the previous heading is `Send text SMS`, this should be `Send binary SMS`

Reply:
Done